domain/infosync/info.go

```diff
@@ -58,7 +58,7 @@ const (
 	// TopologyInformationPath means etcd path for storing topology info.
 	TopologyInformationPath = "/topology/tidb/"
 	// TopologyTimeToLive is ttl for topology.
-	TopologyTimeToLive = 60 * time.Second
+	TopologyTimeToLive = 45
```

Comment:
Please comment why there is no `* time.Second` as above. Please also follow the name convention `TTL` as above.

Reply:
Done